via/services/__init__.py

```diff
@@ -22,39 +22,3 @@ def includeme(config):  # pragma: no cover
     config.register_service_factory(
         "via.services.secure_link.factory", iface=SecureLinkService
     )
-
-
-def create_google_api(settings):
-    """Create from Pyramid settings."""
-
-    return GoogleDriveAPI(
-        credentials_list=load_injected_json(settings, "google_drive_credentials.json"),
-        resource_keys=load_injected_json(settings, "google_drive_resource_keys.json"),
-    )
-
-
-def load_injected_json(settings, file_name):
```

Comment:
Moved to `google_drive.py` and made private there. If this needs to be public I think it belongs to app/config or similar.